### DISCUSSION TITLE
is integration to comfyui possible?

### DISCUSSION THREAD

**Discussion by neonwatty** [neonwatty/meme-search#26] [Ideas]
this would drastically reduce indexing meme's generated by comfy'

suggested by [itshardtopicka_name_](https://www.reddit.com/r/selfhosted/comments/1e3x6l0/comment/ldcm1i4/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button)

**Comment by neonwatty** [comment]
No.

**Comment by matssun** [comment]
**Closed — superseded by ADR-LAIGO-105 (#2114) and ADR-LAIGO-106 (#2115).**

The programmatic optimizer scaffold in Hermes has been replaced by:
- IPromptOptimizer + DSPyPromptOptimizer (ADR-105)
- OutputConstraint types in ILLMClient (ADR-106)

Both implemented in PR #2145 (merged).
